activerecord/test/cases/adapters/abstract_mysql_adapter/warnings_test.rb

```diff
@@ -83,6 +83,8 @@ def setup
       result = @connection.execute("INSERT IGNORE INTO posts (id, title, body) VALUES(#{row_id}, 'Title', 'Body')")
 
       assert_equal [], result.to_a
+    ensure
+      Post.where(id: row_id).delete_all
```

Comment:
This file does not include `require "models/post"`, so we would also need to add that to use `Post`.

But this test seems like the odd one out.  All the other tests use `SELECT 1 + "foo"`, even the `"db_warnings_action allows a list of warnings to ignore"` right above.  So we don't we make this test match the others instead?

```ruby
  test "db_warnings_action allows a list of codes to ignore" do
    with_db_warnings_action(:raise, ["1292"]) do
      result = @connection.execute('SELECT 1 + "foo"')

      assert_equal [1], result.to_a.first
    end
  end
```

Reply:
👍 Changed.